osi_object.proto

```diff
@@ -61,6 +61,22 @@ message StationaryObject
         //
         optional Color color = 4;
 
+        // The IDs of the lanes that the object is assigned to.
+        //
+        // \note Might be multiple if the object streches over multiple lanes.
+        //
+        // \note OSI uses singular instead of plural for repeated field names.
+        //
+        repeated Identifier assigned_lane_id = 5;
+
+        // Percentage value of the object width in the corresponding lane.
```

Comment:
Above a plural is used (_... IDs..._), which makes sense because it is a repeated field. Therefore, I suggest using plural here as well.
```suggestion
        // Percentage values of the object width in the corresponding lane.
```